specs/disk_inventory_etcd_structure.adoc

```diff
@@ -0,0 +1,138 @@
+= Disk inventory etcd structure
+
+This specification specifies the way in which disk inventory has to be structured
+in tendrl's central store and the details that has to be stored in the disk
+inventory
+
+== Problem description
+
+We should have a proper structure to our disk inventory in such a way that we
+store all the needed information in a easily consumable way.
+
+== Use Cases
+
+* This spec provides guidelines about how the disk information is to be
+stored in etcd.
+
+== Proposed change
+
+The disk inventory details will basically stored under 2 sections as follows:
+
+1. /nodes/<node-id>/disks/
+
+* In this location we store all the physical disks available on the node
+with a unique key which is a combination of manufacturer-id+vendor-id+
+device-serial-number this would be unique across tendrl. And this information
+should be available from hwinfo tool.
+
+* under each disk here we have to store all the details about the physical
+  device available via hwinfo tool.
+
+* also the following details have to be stored under each disk(if applicable):
+       - partition details under it
+       - lvm details under
+       - raid specific details
+
+2. /nodes/<node-id>/block-devices/
```

Comment:
id prefer it to be "/nodes/<node-id>/block_devices/"

We dont use hyphen in "list" attributes of objects